website/content/en/docs/olm-integration/legacy/user-guide-package-manifests.md

```diff
@@ -0,0 +1,87 @@
+---
+title: OLM Integration User Guide
+linkTitle: OLM Integration User Guide
```

Comment:
we have 2 docs with the same title which is confusing:


<img width="282" alt="Screenshot 2020-07-02 at 10 29 34" src="https://user-images.githubusercontent.com/7708031/86342166-2193e400-bc4f-11ea-9100-1461d9a59a84.png">

Reply:
Can we aggregated both or change its titles to clarify?
IMO we should have just gone with the name quick-start as well which will cover all steps from A to Z how to integrate the project with OLM.

Reply:
The `legacy/` subdir should have its own sidebar menu item, will fix.

Reply:
the legacy and new ones has the same issue.